src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -310,6 +305,7 @@ public void Unload()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Assembly.Load(byte[], ...) is not supported in AppX.")]
```

Comment:
@AtsushiKan can you remind me, is it correct that Assembly.Load(byte ..) and Assembly.LoadFile will throw PNSE in 2.1 - implementing them is not feasible for some good reason?

if not we should fix them, and hav ean issue # here.